.github/workflows/PR-check-windows.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-2019]
```

Comment:
no linux ,macos ?

Reply:
This is qmake+windows PR check. Will delete this when vcpkg+windows is finished.